doc/TRAINING.rst

```diff
@@ -65,7 +65,7 @@ If you have a capable (NVIDIA, at least 8GB of VRAM) GPU, it is highly recommend
    pip3 uninstall tensorflow
```

Comment:
General comment on this file. Should the github reference on line 126 be re-wored as you did with the other github references?

Reply:
How do you feel about embedding a full listing of the flag definitions in a page?

Reply:
If it's generated from the code, why not.

Reply:
Addressed in the new commit. Built docs: https://community-tc.services.mozilla.com/api/queue/v1/task/Qvo2k34eSDS-GMDQMt5WCg/runs/0/artifacts/public/doc-html.zip